src/App.js

```diff
@@ -1,72 +1,85 @@
+
+import Thankyou from "./components/pages/request/thankYou"
+import RequestVolunteer from "./components/pages/request/reqVolunteer"
+import ReceiverRequest from "./components/pages/request/receiverReq"
+import RequestType from "./components/pages/request/requestType"
+import RequestType1 from "./components/pages/request/requestType1"
+import RequestHelpType from "./components/pages/request/helpType"
+import RequestSpecify from "./components/pages/request/specify"
+import RequestConfirmation from "./components/pages/request/confirmation"
+import VolunteerMessage from "./components/pages/request/volunteerMessage"
+import RequestMap from "./components/pages/request/map"
+
+import HelperMap from "./components/pages/helper/map"
+
+import RatingsPending from "./components/pages/ratings/pending"
+import NavigationStore from "./stores/NavigationStore"
+
+import RegistrationStore from './stores/RegistrationStore'
+import UserStore from './stores/UserStore'
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import "./assets/css/App.css";
+import "./assets/css/media-queries.css";
+
+const stores = {
+  navigation:new NavigationStore(),
+  registration: new RegistrationStore(),
+  user: new UserStore(),
+}
+
+function App() {
+  return (
+    <div className="App">
+      <Provider {...stores}>
+        <BrowserRouter>
+          <Navigation />
+          <Switch>
+            <Route path="/welcome" component={Welcome} />
+            <Route path="/signin" component={Signin} />
+            <Route path="/registration/terms" component={RegistrationTerms} />
+            <Route path="/registration/alternatives" component={RegistrationAlternatives} />
+            <Route path="/registration/signin" component={RegistrationSignin} />
+            <Route path="/registration/information" component={RegistrationInformation} />
+            <Route path="/registration/skills" component={RegistrationSkills} />
+            <Route path="/registration/finish" component={RegistrationFinish} />
+	    <Route path="/request/helpType" component={RequestHelpType} />
+	    <Route path="/request/thankYou" component={Thankyou} />
+            <Route path="/request/reqVolunteer" component={RequestVolunteer} />
+	    <Route path="/request/receiverReq" component={ReceiverRequest} />
+	    <Route path="/request/requestType" component={RequestType} />
+	    <Route path="/request/requestType1" component={RequestType1} />
+            <Route path="/request/specify" component={RequestSpecify} />
+            <Route path="/request/confirmation" component={RequestConfirmation} />
+	    <Route path="/request/volunteerMessage" component={VolunteerMessage} />
+            <Route path="/request/map" component={RequestMap} />
+            <Route path="/helper/map" component={HelperMap} />
+	    <Route path="/ratings/pending" component={RatingsPending} />
+	    <Route path="/stores/NavigationStore" component={NavigationStore} />

```

Comment:
@sunivihan the NavigationStore should not be added here, it's added via the stores property above as it's a non-visual component, thanks !

Reply:
And I think there's some conflict to be resolved with this file and master still, I'm not sure why the diff isn't picking line changes, editor settings / newlines, maybe ?